cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -234,9 +251,16 @@ func gitIndex(c gitIndexConfig, o *indexArgs, sourcegraph Sourcegraph, l sglog.L
 			"-C", gitDir,
 			"-c", "protocol.version=2",
 			"-c", "http.extraHeader=X-Sourcegraph-Actor-UID: internal",
-			"fetch", "--depth=1", o.CloneURL,
+			"fetch", "--depth=1", "--no-tags",
 		}
 
+		// If there are no exceptions to MaxFileSize (1MB), we can avoid fetching these large files.
+		if len(o.LargeFiles) == 0 {
```

Comment:
Does SG populate anything here by default?

Reply:
No it does not. It requires a customer to modify the site config.